### PR TITLE
Updated marketplace in the Portal

### PR DESCRIPTION
**Pull request by garcia-alejandro**
# Description 
* Added the last changes of the Marketplace on the Portal application
* Modified the behavior of the Private file component
* Refactoring the flow of the custodian when the extrinsic is called
* Removed classes from Afloat repo
## Tickets
* #17 
* #20

### CODE REVIEW

File: src/components/marketplace/marketplace-list.vue [hunk 1]
```diff
@@ -5,14 +5,12 @@
   #items(v-else)
     .row.justify-start.q-px-md
       .col-12
-        q-input(
+        t-input(
```
Comment (jmgayosso) [inline src/components/marketplace/marketplace-list.vue]: why we are using t-input instead q-input???

File: src/css/colors.styl [hunk 2]
```diff
@@ -5,7 +5,7 @@ $black = #12002C
 $grayDark = #C1C1C1
 $accent = #30E6FC
 $grayLigth = #FFFFFF
-$bg-body   = rgba(0,0,0,.25)
```
Comment (jmgayosso) [inline src/css/colors.styl]: this should be the right color, please return to it

File: src/css/quasar.variables.sass [hunk 3]
```diff
@@ -24,7 +24,7 @@ $negative  : #C10015
 $info      : #31CCEC
 $warning   : #F2C037
 
-$bg-body: rgba(0,0,0,.25)
```
Comment (jmgayosso) [inline src/css/quasar.variables.sass]: this should be the right color, please change it

File: src/css/colors.styl (hunk 2 above)
Comment (garcia-alejandro) [reply src/css/colors.styl]: Got it!

File: src/mixins/validation.js [hunk 4]
```diff
@@ -21,7 +21,9 @@ export const validation = {
         isValidDerivationPath: val => /([m]\/[0-9]{1,2}(')(\/)[0-9]{1,2}(')\/[0-9]{1,2}(')\/[0-9]('){1,30})/.test(val) || this.$t('forms.errors.isNotValidDerivationPath'),
         isValidMainetBTC: val => /([13][a-km-zA-HJ-NP-Z1-9]{25,34}|bc1[ac-hj-np-zAC-HJ-NP-Z02-9]{11,71})/.test(val) || this.$t('forms.errors.isNotValidMainetBTC'),
         isValidPolkadotAddress: val => this.$store.$polkadotApi?.isValidPolkadotAddress(val) || this.$t('forms.errors.isNotValidPolkadotAddress'),
-        notOwnAccount: value => val => !(value === val) || this.$t('forms.errors.notOwnAccount')
+        notOwnAccount: value => val => !(value === val) || this.$t('forms.errors.notOwnAccount'),
+        greaterOrEqualThanString: value => val => val.length >= value || this.$t('forms.errors.greaterOrEqualThanString', { value }),
```
Comment (jmgayosso) [inline src/mixins/validation.js:25]: why we are duplicating this rule?

File: src/pages/marketplace/custodian/index.vue [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+          })
+          return applicant
+        })
+      } catch (error) {
+        console.error('error', error)
+        tmpApplicants = applicants
+      }
+      return tmpApplicants
+    },
+    async loginUser () {
```
Comment (jmgayosso) [inline src/pages/marketplace/custodian/index.vue]: I think we should refactor this, the portal is handling this part currently

File: src/pages/marketplace/index.vue [hunk 6]
```diff
@@ -134,13 +133,26 @@ export default {
...
+        this.showNotification({ message: 'You must be logged in to create a marketplace', color: 'negative' })
+      }
     }
   }
 }
 </script>
 
 <style lang="stylus" scoped>
-@import '~/css/colors.styl'
+@import '~/css/app.styl'
```
Comment (jmgayosso) [inline src/pages/marketplace/index.vue:150]: why we are importing all app.styl instead just colors???

**Review by jmgayosso — COMMENTED** [review]
I have Some question and I think siome actions should be done first or TODO, please review

File: src/components/marketplace/marketplace-list.vue (hunk 1 above)
Comment (garcia-alejandro) [reply src/components/marketplace/marketplace-list.vue]: Just for the Style. I can add the styles manually like Outlined. So I'll use q-input

File: src/mixins/validation.js (hunk 4 above)
Comment (garcia-alejandro) [reply src/mixins/validation.js:25]: My bad. Maybe I added when fixing the conflicts. I'll remove the duplicate line. Thanks!

File: src/css/quasar.variables.sass (hunk 3 above)
Comment (garcia-alejandro) [reply src/css/quasar.variables.sass]: Got it! thanks!

File: src/mixins/validation.js (hunk 4 above)
Comment (garcia-alejandro) [reply src/mixins/validation.js:25]: Sorry, about this, greaterOrEqualThanString only applies for `String` the other is used on `Numbers`, that's the difference

Comment (jmgayosso) [reply src/mixins/validation.js:25]: > Sorry, about this, greaterOrEqualThanString only applies for `String` the other is used on `Numbers`, that's the difference

So we are using this rules to diff the length for a string? why is this need?

File: src/components/marketplace/marketplace-list.vue (hunk 1 above)
Comment (jmgayosso) [reply src/components/marketplace/marketplace-list.vue]: Ok I got it, So I think we should use a new component called h-input (hashed input), this should allow us encapsulate the styles for all project in each different project, what do you think?

Comment (garcia-alejandro) [reply src/components/marketplace/marketplace-list.vue]: I agree

**Review by jmgayosso — APPROVED** [review]

**Pull request merged**